youtube_dl/YoutubeDL.py

```diff
@@ -833,6 +833,8 @@ def iter_playlistitems(format):
                                                       extra_info=extra)
                 playlist_results.append(entry_result)
             ie_result['entries'] = playlist_results
+            if result_type == 'playlist':
+                self.to_screen('[info] Playlist download process finished')
```

Comment:
Is [download] relevant if the whole process finishes after an [ffmpeg] or [avconv] conversion?